apps/prairielearn/assets/stylesheets/pageLayout.css

```diff
@@ -360,3 +363,11 @@ body:has(div.app-main-container) {
     transition: none;
   }
 }
+
+.page-footer-text {
+  color: rgb(5.2, 44, 101.2);
+}
+
+.container.min-h-100 {
+  min-height: calc(100% - var(--navbar-height));
+}
```

Comment:
I feel like this is overcomplicating things. I don't actually mind if the footer is visible without scrolling on the homepage.

Reply:
I actually think it is harder to show the footer like this, unless you want to put all content of the homepage in a flexbox.